src/cable/handler.cr

```diff
@@ -37,12 +37,13 @@ module Cable
 
         socket.on_close do
           connection.close
-          Cable::Logger.info "Finished \"#{path}\" [WebSocket] for #{remote_address} at #{Time.now.to_s}"
+          Cable::Logger.info "Finished \"#{path}\" [WebSocket] for #{remote_address} at #{Time.utc.to_s}"
         end
       end
 
       Cable::Logger.info "Successfully upgraded to WebSocket (REQUEST_METHOD: GET, HTTP_CONNECTION: Upgrade, HTTP_UPGRADE: websocket)"
       content = ws.call(context)
+      content.as(Proc(IO, Nil)).call(context.response.output)
```

Comment:
I'm not sure if this is needed, but without it, the websocket block is never called (at least on this Fiber). By adding it, the `socket.send` welcome is added to the response.

One other thing is that the type of `ws.call(context)` kept coming back as 

```
In src/cable/handler.cr:48:21

 48 | raise content.call.inspect
                    ^---
Error: undefined method 'call' for Array(Log::Entry) (compile-time type is (Array(Log::Entry) | Bool | IO | Int32 | Int64 | Proc(IO, Nil) | Nil))
```

so I had to typecast it just to get this to run.